### PR TITLE
Add new ci pipeline: helm chart end to end testing

### PR DESCRIPTION
**Pull request by iskhakov**
**What this PR does**:
This PR add the GHA pipeline for integration test of the helm chart and instruction how to run the chart locally.
Check [.github/workflows/helm_tests.yml](https://github.com/grafana/oncall/pull/855#diff-9deeb3a784538a09440a6c119a6876b576e0062e3165ee4140dd6967d64cfb4e) file comments and step titles 

**Which issue(s) this PR fixes**:

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated

### CODE REVIEW

File: .github/workflows/helm_tests.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              "image_url": "https://upload.wikimedia.org/wikipedia/commons/e/ee/Grumpy_Cat_by_Gage_Skidmore.jpg",
+              "state": "alerting",
+              "link_to_upstream_details": "https://en.wikipedia.org/wiki/Downtime",
+              "message": "Smth happened. Oh no!"
+            }'
+
+      # Running k6 on the local environment
+      # Commenting out because doesn't work for the local server
+      # https://github.com/grafana/k6-action#side-by-side-with-the-system-under-test
+#      - name: Install k6
```
Comment (Matvey-Kuk) [inline .github/workflows/helm_tests.yml:71]: May be remove in this case?

**Review by Matvey-Kuk — APPROVED** [review]

File: helm/.jshintrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "esversion": 6
+}
```
Comment (joeyorlando) [inline helm/.jshintrc:3]: what is this file?

File: .github/workflows/helm_tests.yml (hunk 1 above)
Comment (iskhakov) [reply .github/workflows/helm_tests.yml]: done

**Review by joeyorlando — APPROVED** [review]

File: helm/.jshintrc (hunk 2 above)
Comment (iskhakov) [reply helm/.jshintrc]: it was for k6, which I just removed

**Comment by joeyorlando** [comment]
@iskhakov do you want to merge this before https://github.com/grafana/oncall/pull/624 or vice-versa?

**Review by joeyorlando — APPROVED** [review]

**Pull request merged**